### PR TITLE
Provide stubs

### PR DESCRIPTION
**Pull request by codedge**
Create exercise stubs, see #347 

This is just the first set, I need to go through a couple of others. It seems to be the minority that have stubs.

### CODE REVIEW

**Comment by codedge** [comment]
I ran through all the exercises adding stubs. Should be good now.

**Comment by neenjaw** [comment]
@codedge - Thanks for opening this PR! I'll review this tonight. From a brief skim, this looks great!

File: exercises/practice/accumulate/Accumulate.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-function accumulate(array $input, callable $accumulator)
+function accumulate(array $input, callable $accumulator): array
 {
-    // YOUR CODE GOES HERE
+    \BadFunctionCallException("Implement the accumulate function");
```
Comment (neenjaw) [inline exercises/practice/accumulate/Accumulate.php:5]: Shouldn't these throw the error?
```suggestion
    throw new \BadFunctionCallException("Implement the accumulate function");
```

**Review by neenjaw — CHANGES_REQUESTED** [review]

File: exercises/practice/accumulate/Accumulate.php (hunk 1 above)
Comment (codedge) [reply exercises/practice/accumulate/Accumulate.php]: Holy... your are completely right. Fixed for all the tests.

**Review by neenjaw — APPROVED** [review]
Very nice! Awesome work @codedge!

**Pull request merged**